src/pydata_sphinx_theme/assets/styles/sections/_sidebar-primary.scss

```diff
@@ -207,11 +201,12 @@ nav.bd-links {
     }
   }
 
+  // Hover should still work on top of the active link
```

Comment:
this works on the text, but not on the div-edge color:

![Screenshot_2023-06-26_12-06-13](https://github.com/pydata/pydata-sphinx-theme/assets/1810515/7fb23838-29c0-4e2a-9b98-5ba269f3b8d9)

you can't see my cursor in the screenshot, but (perhaps obviously) it's on the "theme variables and CSS" TOC item

Reply:
That is intentional - in menus, the hover, active and focus states should not interfere with the current state (i.e. a menu item could be focused with a keyboard and retain its location/current indicator simultaneously). So in the sidebar:

* The teal bar [indicates the current location within the sidebar](https://www.w3.org/TR/WCAG20-TECHS/G128)
* The hover should be displayed alongside the current state (n[ot override it](https://www.w3.org/WAI/tutorials/menus/styling/#hover-and-focus-states) 

But as I write this, I realise that this is directly violating the `not rely on colour` criteria - so I might need to think about how to convey this instead (in terms of the thickness of the sidebar or similar)

Reply:
Just pushed an update - the `hover` style (thick underline) is applied on top of the `current` one.

Reply:
ok I'm happy with that solution!